src/signals/incident/components/form/MapSelectors/Caterpillar/CaterpillarLayer/CaterpillarLayer.tsx

```diff
@@ -94,17 +93,33 @@ export const CaterpillarLayer: FC = () => {
         />
       )
     },
-    [
-      meta.extraProperties,
-      meta.featureTypes,
-      meta.icons,
-      removeItem,
-      selection,
-      setItem,
-    ]
+    [meta.extraProperties, meta.featureTypes, removeItem, selection, setItem]
   )
 
-  return <>{features.map(getMarker)}</>
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const reportedFeatureType = meta.featureTypes.find(
+    ({ typeValue }) => typeValue === 'reported'
+  )!
```

Comment:
Same applies here and below; maybe extend the `Meta` type for this specific implementation?